Scripts/Runtime/CallbackHandlers/MultiValueHandler.cs

```diff
@@ -73,15 +73,26 @@ protected override void OnHandleResponse(WitResponseNode response)
                         formatEvent.onFormattedValueEvent?.Invoke(result);
                     }
                 }
+            }
 
-                List<string> values = new List<string>();
-                for (int i = 0; i < references.Length; i++)
+            // We want to invoke if at least one value was found. Anything that
+            // was missing will be empty, but we must have at least one value
+            // in the response for the invoke to happen.
```

Comment:
Does Wit consider something to be a match if one part of it matches and the others are empty? Should we have "required" and "non-required" parameters? This change here seems like a nontrivial change of meaning.

Reply:
This was the intended meaning of this callback for this utility component. As long as one entity returns a value you get the callback. The array structure is always the same. The true match is on the intent, so maybe I should have this callback fire no matter what if the intent is matched and let the dev decide what to do with no entities.